Fix inconsistencies for the tree view interface (#21013)
directus/directus: .changeset/rotten-goats-clap.md

---
'@directus/app': patch
---

> Fixed visual insistencies for the tree view interface. Replaced `ul/li` usage with `v-list/v-list-item`. Added skeleton loader

Reviewer: ```suggestion
Fixed visual inconsistencies in the tree view interface
```
